ATAPHtmlReport/CIS_Microsoft_Windows_10_Enterprise_Release_21H1_Benchmark_v1-MITRE ATT&CK Mappings.json

```diff
@@ -0,0 +1,10574 @@
+[
```

Comment:
da jetzt alles in der json steht kann man die alte excel rausloeschen

Reply:
Excel-Datei wurde gelöscht